pyhf/commandline.py

```diff
@@ -234,10 +240,19 @@ def cls(workspace, output_file, measurement, patch, testpoi, teststat):
         patches=patches,
         modifier_settings={'normsys': {'interpcode': 'code4'}},
     )
+
+    optconf = {k: v for item in optconf for k, v in item.items()}
+
+    # set the new optimizer
+    if optimizer:
+        new_optimizer = getattr(optimize, optimizer)
+        set_backend(tensorlib, new_optimizer(**optconf))
+
     result = hypotest(testpoi, w.data(p), p, qtilde=is_qtilde, return_expected_set=True)
     result = {'CLs_obs': result[0].tolist()[0], 'CLs_exp': result[-1].ravel().tolist()}
+
     if output_file is None:
-        click.echo(json.dumps(result, indent=4, sort_keys=True))
+        print(json.dumps(result, indent=4, sort_keys=True))
```

Comment:
@kratsg Just curious why this was changed to `print`. Just better formatting?

Reply:
Oh, no. it should've been fixed...